llama_index/indices/multi_modal/base.py

```diff
@@ -85,7 +85,7 @@ def image_vector_store(self) -> VectorStore:
         return self._image_vector_store
 
     @property
-    def image_embed_model(self) -> EmbedType:
+    def image_embed_model(self) -> MultiModalEmbedding:
```

Comment:
@nerdai basically I changed this to MM embedding model.
As discussed offline, we will have a dedicated image_to_text_embed_model separately

Reply:
Got it, thanks @hatianzhang! Will check how it affects my MM RAG Eval nb and make a PR for any necessary changes.